### ISSUE TITLE
[New Repository] - Osclass Enterprise

### ISSUE THREAD

**Issue by MercanoGlobal** [osclass/Osclass#2331]
If anyone is still interested in using Osclass, please note that project is continued here:
https://github.com/MercanoG/Osclass-Enterprise

**Osclass Enterprise** fixed most of the old Osclass issues and added lots of new features, while keeping the compatibility with all plugins and themes.
It's what Osclass should have been in the first place, in terms of features.

**Comment by erobertsme** [comment]
How does this differ from https://github.com/mindstellar/Osclass ?

**Comment by MercanoGlobal** [comment]
@omfgtora **Osclass Enterprise** has nothing in common with **Mindstellar**.

- Mindstellar is a complete Osclass rewrite, which changed/broke the entire system by refactoring almost every function.
While an admirable effort, it created major incompatibilities with existing plugins and themes and introduced some new bugs, basically becoming a release that could only be used by other developers as a tech demo.
It shouldn't be used in production under any circumstance!

- Osclass Enterprise is the continuation of the Osclass project - so basically we forked the last release (v3.8.0) and we did the following:

1. Fixed every bug we found and also all that were reported anywhere on the web (forums, other repositories etc.).
2. Enhanced the overall security (hacking/spam/bot protection).
3. Added lots of new features ([Changelog](https://github.com/MercanoG/Osclass-Enterprise/blob/main/CHANGELOG.txt)).
4. Added PHP 8+ compatibility (Mindstellar doesn't even support PHP 7.4 properly).

So in other words, [Osclass Enterprise](https://github.com/MercanoG/Osclass-Enterprise) is the same Osclass we all used and loved, but with most of its issues fixed and with a lot of powerful enhancements, that make it a perfect solution for Enterprise users (users that need something stable to build their business on).
Our unofficial motto for this release is "_Install and forget_". Meaning that once installed and configured, you shouldn't have to do anything else to your setup for it to run perfectly (e.g. updates that may break stuff, core changes, worrying about PHP incompatibilities etc.).

I hope this answers your question.
Regards!

**Comment by erobertsme** [comment]
Just a follow up on this. I've since tried this version and I found it to be the best version of OSClass. I have now switched over my instance to OSClass Enterprise and it has proven to be much faster and more reliable. I highly suggest OSClass Enterprise as the successor.
